lagom-descriptor-generator-sbt-plugin/src/sbt-test/plugin/petstore-java/test

```diff
@@ -0,0 +1,11 @@
+# Structure of this test:
+# =======================
+
+# Here we test that the swagger-to-java lagom descriptor and model generators create files on compilation time without
+# further intervention. The contents are not asserted
+
+> clean
+> compile
+$ exists petstore-api/target/scala-2.11/openapi/main/java/com/example/PetApi.java
+$ exists petstore-api/target/scala-2.11/openapi/main/java/com/example/StatusEnum.java
+$ exists petstore-api/target/scala-2.11/openapi/main/java/com/example/Pet.java
```

Comment:
This `scripted` test uses the generator in a `petstore-api`. That's a team working on the producer end of the API.